coderedcms/bin/coderedcms.py

```diff
@@ -62,32 +42,48 @@ def run(self, project_name=None, dest_dir=None):
                      "Python module and cannot be used as a project "
                      "name. Please try another name." % project_name)
 
-        print("Creating a CodeRed CMS project called %(project_name)s" % {'project_name': project_name})  # noqa
-
-        # Create the project from the Wagtail template using startapp
+        # Create a random SECRET_KEY to put it in the main settings.
+        options['secret_key'] = get_random_secret_key()
 
-        # First find the path to Wagtail
+        # add custom args
         import coderedcms
         codered_path = os.path.dirname(coderedcms.__file__)
         template_path = os.path.join(codered_path, 'project_template')
-
-        # Call django-admin startproject
-        utility_args = ['django-admin.py',
-                        'startproject',
-                        '--template=' + template_path,
-                        '--ext=html,rst',
-                        '--name=Dockerfile',
-                        project_name]
-
-        if dest_dir:
-            utility_args.append(dest_dir)
-
-        utility = ManagementUtility(utility_args)
-        utility.execute()
-
-        print("Success! %(project_name)s has been created" % {'project_name': project_name})  # noqa
-
-
+        options['template'] = template_path
+        options['extensions'] = ['py', 'html', 'rst', 'md']
+        options['files'] = ['Dockerfile']
+
+        # Set default options and print a friendly message
+        output = "Creating a CodeRed CMS project called %(project_name)s"
+        if options.get('sitename'):
+            output += " for %(sitename)s"
+        else:
+            options['sitename'] = project_name
+        if options.get('domain'):
+            output += " (%(domain)s)"
+        else:
+            options['domain'] = 'localhost'
+        print(output % {
+            'project_name': project_name,
+            'sitename': options.get('sitename'),
+            'domain': options.get('domain'),
+        })
+
+        # Run command
+        super().handle('project', project_name, target, **options)
+
+        # Be a friend once again.
+        print("Success! %(project_name)s has been created" % {'project_name': project_name})
+
+        nextsteps = """
+Next steps:
+    1. cd %(directory)s/
```

Comment:
Did it actually do that for you? If so that would be a bug. If directory is not provided, it should use normal startproject behavior and create a directory named {{ project_name }}

Reply:
Oh, yea that makes sense then.